packages/datetime/test/dateInputTests.tsx

```diff
@@ -14,6 +14,14 @@ import { padWithZeroes } from "../src/common/utils";
 import { Classes, DateInput } from "../src/index";
 
 describe("<DateInput>", () => {
+    it("handles null inputs without crashing", () => {
+        mount(<DateInput value={null} />);
```

Comment:
That should be implicit in the tests; we shouldn't allow uncaught exceptions. If that's not the case we should fix that soon...